app-a/ch07/troubleshooting-pod/web-app/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM node:12
+FROM node:latest
```

Comment:
This should not use `latest`. It should use a concrete version.

Reply:
done in https://github.com/bmuschko/cka-study-guide/pull/16